docs/UserGuide.md

```diff
@@ -487,17 +495,32 @@ Format: `view t/ENTRY_TYPE mo/MONTH`
 
 1. View a Line Graph of all expenditures in a specified month
     - Examples: `view t/e mo/2022-01` where we specify the month to be January 2022.
-    - Expected: `Show graphically all expenditure by month` and a Line Graph on the right of the application
+    - Expected: `Show graphically all expenditure by month` and a Line Graph on the right of the application.
+   The List Display will also be filtered to show only expenditures in Jan 2022.
 
    ![ViewCommandExpenditureMonth](images/ug/ViewCommandExpenditureMonth.png)
 
 2. View a Line Graph of all incomes in a specified month
     - Examples: `view t/i mo/2022-01`
     - Expected: `Show graphically all income by month` and a Line Graph on the right of the application
+    The List Display will also be filtered to show only income in Jan 2022.
+
+<div markdown="span" class="alert alert-info">:information_source: **Info:**
+The `MONTH` field is optional. If a month is specified, the entry details for the corresponding month is shown,
+accompanied by a Line Graph on the right of the application. If no month is specified, the application displays an
+error.
+</div>
```

Comment:
I think this might be confusing since month field is optional it shouldn't display an error if no month is specified

Reply:
this might be a remnant from when there was a g/ identifier